aiohttp/helpers.py

```diff
@@ -40,7 +40,7 @@
     cast,
 )
 from urllib.parse import quote
-from urllib.request import getproxies
+from urllib.request import getproxies, proxy_bypass  # noqa: F401
```

Comment:
Is there a reason to import this here, rather than just import it directly in client.py?

Reply:
It looked like this is where it belonged, it's where all the helpers are. And it looked like this file was meant to expose helper functions, so if in the future a new implementation of proxy_bypass was written only the code here would need to be changed. But I could be wrong, and I'll move it out that is not the case?

Reply:
Maybe somebody else has an opinion on it, but if we're just using something from the standard library, I'm not sure there's much benefit in re-exporting it (e.g. the other imports like getproxies are used within this file, not exported elsewhere).

It just jumped out at me as a little odd (which is only highlighted by the noqa). If there's the possibility that we might want to replace this function in the future with a different implementation, then perhaps it's reasonable.